web/public/elm-setup.js

```diff
@@ -62,8 +62,7 @@ app.ports.renderPipeline.subscribe(function (values) {
   const jobs = values[0];
   const resources = values[1];
   renderingModulePromise.then(({renderPipeline}) =>
-    // elm 0.17 bug, see https://github.com/elm-lang/core/issues/595
-    setTimeout(() => renderPipeline(jobs, resources, app.ports.newUrl), 0)
```

Comment:
we should investigate if this ELM bug is still relevant... https://github.com/elm/core/issues/595

Reply:
so this is still relevant on Safari, doesn't render without the animationFrame

Reply:
Yes I could reproduce the issue with Safari 18.5. It rendered a blank screen. But worked fine on Firefox/Chrome. Not sure what I did to reproduce it. Will need to try again.